fhir-persistence/src/main/java/com/ibm/fhir/persistence/interceptor/FHIRPersistenceEvent.java

```diff
@@ -21,45 +17,19 @@
  * This class represents an event fired by the FHIR persistence interceptor framework.
  */
 public class FHIRPersistenceEvent {
```

Comment:
Leaving a general comment... so ... I do think we're locking down the flexibility of downstream applications to source custom headers and custom rest behaviors from the URL.  What is the approach in this case? 

For instance, we have the DataSourceID...

Reply:
We already have that dataStoreId (and others) in the FHIRRequestContext which is a threadlocal that we set up in the FHIRRestServlet.  My thought is that we can put context related to the HTTP request in there as needed, but its really just a thought...

Reply:
I would think maybe there there are more info in the header than what we plan to use now, maybe we can find values in the other headers in the future analytic, so I would think not a bad idea to keep the headers ...

Reply:
I agree with Albert

Reply:
I've updated the PR to capture httpHeaders in the FHIRRequestContext